mailbox/event/event-memory/src/main/java/org/apache/james/mailbox/events/InVMEventBus.java

```diff
@@ -53,6 +55,7 @@ public InVMEventBus(EventDelivery eventDelivery, RetryBackoffConfiguration retry
         this.eventDeadLetters = eventDeadLetters;
         this.registrations = Multimaps.synchronizedSetMultimap(HashMultimap.create());
         this.groups = new ConcurrentHashMap<>();
```

Comment:
You probably lost a fixup but you probably don't need a ConcurrentHashMap because you can only write once to this field.